web/profiles/joinup/config/install/search_api.index.collections.yml

```diff
@@ -0,0 +1,73 @@
+uuid: ac8e6731-1174-4f27-ab00-de719168b34b
```

Comment:
Don't commit the UUID. Are you using the Config Devel module to export config? That should take care of this automatically.

This config item should be added to the list in `joinup.info.yml`.